api/proto/teleport/accesslist/v1/accesslist.proto

```diff
@@ -156,3 +156,43 @@ enum IneligibleStatus {
   // Only applicable to members.
   INELIGIBLE_STATUS_EXPIRED = 4;
 }
+
+// Review is a review of an access list.
+message Review {
+  // header is the header for the resource.
+  teleport.header.v1.ResourceHeader header = 1;
+
+  // spec is the specification for the access list review.
+  ReviewSpec spec = 2;
+}
+
+// ReviewSpec is the specification for an access list review.
+message ReviewSpec {
+  // access_list is the name of the access list that this review is for.
+  string access_list = 1;
+
+  // review is the user who performed the review.
+  string reviewer = 2;
+
+  // review_date is the date that this review was created.
+  google.protobuf.Timestamp review_date = 3;
+
+  // body is an optional plaintext attached to the review that can be used by the review for arbitrary
```

Comment:
i wanted to understand this field more. is this something the reviewer can provide? and if so, what's an example?

Reply:
I think the idea here is if the reviewer wanted to add any sort of plaintext here, they could. So something like,

```
Found 12 unnecessary users.
Tuned membership requirements.
```

This is likely unnecessary, however.

Reply:
oh, then, should the field be named like `note` or `review_note`? b/c `body` sounds confusing to m

Reply:
That's way better, agreed. I'll rename!